tests/check.sh

```diff
@@ -3,6 +3,18 @@ set -euo pipefail
 dn=$(dirname "$0")
 srcdir=$(cd "${dn}"/.. && pwd)/src
 
+# We want to make use of the `-x|--external-sources` flag, which was made
+# available in v0.4.0.  So check for the flag and disable the use of
+# ShellCheck if not present.
+set +eo pipefail
+shellcheck |& grep -q -- --external-sources
+HASSHELLCHECK=$?
+set -eo pipefail
+
+if [[ ${HASSHELLCHECK} -ne 0 ]]; then
+    echo "WARNING: available ShellCheck is too old. Shell script checking is disabled."
```

Comment:
shellcheck is too old, or not available ?

Reply:
`ShellCheck` is included in `src/deps.txt`, so it **should** always be there (and we don't plan on removing it from the deps).  I can update the message if it is preferred.